src/app/shared-services/inferences/engine/definitions.ts

```diff
@@ -5,9 +5,13 @@ import { InferenceDefinition, InferenceType } from './types'
 export const WorkInference = new InferenceDefinition(
   'workplace',
   InferenceType.work,
+  'business',
   (lang?: string) => 'Workplace',
-  (r: Inference, lang?: string) =>
-    `We assume your workplace is at ${r.lonLat} with a confidence of ${r.confidence}.`,
+  (r: Inference, lang?: string) => {
+    const latLng = `${r.latLng[0].toFixed(2)}, ${r.latLng[1].toFixed(2)}`
```

Comment:
`toFixed(2)` will result in GPS values with an accuracy of around 1km. If we want to have a more precise output we might need to add some decimals

Reply:
I was going for an "aesthetically pleasing" textual representation of the inferences, but indeed the accuracy of those values is not high enough to make any conclusions. I think I the long run we would present more intuitive information - e.g. an address instead of geocoordinates.